### PR TITLE
Use different L1 & L2 deployers

### PR DESCRIPTION
**Pull request by telome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,32 +71,38 @@ L2=arbitrum_one_sepolia
...
+forge script script/Init.s.sol:Init --sender $L1_DEPLOYER --private-key $L1_PRIVATE_KEY --slow --multi --broadcast
 ```
 
 ### Test the deployment
 
 Make sure the deployer account holds at least 10^18 units of the first token listed under `"l1Tokens"` in `script/output/{chainId}/deployed-latest.json`. To perform a test deposit of that token, use the following command:
 
 ```
-forge script script/Deposit.s.sol:Deposit --sender $DEPLOYER --private-key $PRIVATE_KEY --slow --multi --broadcast
+forge script script/Deposit.s.sol:Deposit --sender $L1_DEPLOYER --private-key $L1_PRIVATE_KEY --slow --multi --broadcast
```
Comment (sunbreak1211) [inline README.md]: I haven't run the scripts yet, but I think we should also modify Deposit and Withdraw, first one to deposit in the L2 address, second to withdraw to the L1 address (this last one is probably subjective).

**Comment by oldchili** [comment]
Just a note that it seems foundry supports loading the private key in the startBroadcast command itself - 
https://book.getfoundry.sh/cheatcodes/start-broadcast

So there is an option to still use one script, and just load the private keys. An advantage here is that you don't need to assume the l2 nonce is not touched after pre-computing it.
And also probably a smaller change and less duplicated code.
Not feeling too strong either way.

**Comment by telome** [comment]
> Just a note that it seems foundry supports loading the private key in the startBroadcast command itself - https://book.getfoundry.sh/cheatcodes/start-broadcast
> 
> So there is an option to still use one script, and just load the private keys. An advantage here is that you don't need to assume the l2 nonce is not touched after pre-computing it. And also probably a smaller change and less duplicated code. Not feeling too strong either way.

Nice, didn't know about this option. Will try to rewrite the script using it then.

File: .env.example [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+export FOUNDRY_SCRIPT_DEPS=deployed
+export FOUNDRY_EXPORTS_OVERWRITE_LATEST=true
+export L1="sepolia"
+export L2="arbitrum_one_sepolia"
+export ETH_RPC_URL=
+export ARBITRUM_ONE_RPC_URL=
+export SEPOLIA_RPC_URL=
+export ARBITRUM_ONE_SEPOLIA_RPC_URL=
+export L1_DEPLOYER=
+export L2_DEPLOYER=
```
Comment (sunbreak1211) [inline .env.example]: The way it is done we don't really need these ones.

**Review by sunbreak1211 — APPROVED** [review]
LGTM

**Review by oldchili — APPROVED** [review]
LGTM

**Pull request merged**